### PR TITLE
New header design

### PR DESCRIPTION
**Pull request by jarednorman**

### CODE REVIEW

**Comment by Sinetheta** [comment]
Screenshot plaz

**Comment by jarednorman** [comment]
whateverkevin

**Comment by Sinetheta** [comment]
Looks hot. Might want a "responsive table" layout of some kind, but I really dig the new header.

**Comment by jarednorman** [comment]
PR's accepted :trollface:, but. yeah, We've done that in other tables. By we I mean @graygilmore. It's a good idea.

**Comment by graygilmore** [comment]
The green of the navigation is a little in-your-face. What about something like:

screen shot 2016-02-22 at 11 06 16 pm

**Comment by graygilmore** [comment]
Really slick change, I like!

File: app/assets/stylesheets/_base.scss [hunk 1]
```diff
@@ -12,13 +11,13 @@ select {
 a {
   &:link,
   &:visited {
-    color: $green;
+    color: $blue;
```
Comment (graygilmore) [inline app/assets/stylesheets/_base.scss:14]: This makes me trés sad. No `$color-link` and `$color-link-hover`?

**Comment by jarednorman** [comment]
But ping pong tables are green!

File: app/views/layouts/_header.html.haml [hunk 2]
```diff
@@ -1,16 +1,26 @@
-%header.main-header
-  %nav.secondary-navigation
-    - unless write_access?
+%nav.main-navigation{ class: [write_access? && "has-write-access"] }
+  .main-header-wrap
+    %header.main-header
+      %h1.site-title
```
Comment (graygilmore) [inline app/views/layouts/_header.html.haml:4]: I believe this should only be the `h1` on the home page and be a `div` everywhere else.

**Comment by graygilmore** [comment]
> But ping pong tables are green!

Our table is blue. You haven't been gone that long!

**Comment by jarednorman** [comment]
Hahaha

**Comment by jarednorman** [comment]
Yeah, looks better in the blue I agree.

File: app/assets/stylesheets/_base.scss (hunk 1 above)
Comment (adammathys) [inline app/assets/stylesheets/_base.scss:14]: > très

Nobody uses old or middle French anymore.

Comment (adammathys) [inline app/assets/stylesheets/_base.scss:14]: https://github.com/porkchopclub/porkchop/issues/11

**Pull request merged**